exec/java-exec/src/main/resources/rest/query/query.ftl

```diff
@@ -87,13 +87,30 @@
   </form>
 
   <script>
+    // Remember form field values over page reloads
+    $('input[type=text],input[type=checkbox],input[type=radio],select').each(function() {
+       var $input = $(this);
+       var key = 'saved_query_' + $input.attr('name');
+       var savedValue = sessionStorage.getItem(key);
+       if($input.attr("type") === 'checkbox') {
+         if(savedValue === 'true') $input.prop('checked', true);
```

Comment:
```suggestion
         $input.prop('checked', savedValue === 'true');
```
This should work.

Reply:
I don't want to check the prop unless there is actually a valid value set in the session storage.  If the value is `null` and there's some other process that sets a default, we should keep that default (e.g. not uncheck a checkbox that would be checked by default).